Lesson02-GitHub-Repo-Search/T02.01-Exercise-CreateLayout/app/src/main/res/layout/activity_main.xml

```diff
@@ -19,15 +19,16 @@
 <!--TODO (2) Make the orientation vertical-->
 <!--TODO (3) Give left, right, and top padding of 16dp-->
 <!--TODO (4) Remove the line that declares the id, we don't need it-->
-<!--TODO (5) Remove the xmlns:app declaration, we don't need that anymore-->
+
```

Comment:
Why is purpose of this line break?

Reply:
I was thinking this will create a logical separation between the todos. It's going from todo 4 to todo 6.